packages/mobile/src/home/WalletHome.tsx

```diff
@@ -46,6 +47,7 @@ interface StateProps {
   numberVerified: boolean
   cashInButtonExpEnabled: boolean
   balances: Balances
+  showTokensinHome: boolean
```

Comment:
nit: typo `showTokensinHome` -> `showTokensInHome`

Reply:
hm not sure what you mean here they are the same haha

Reply:
It's the capitalization in the i showTokens **i** nHome vs showTokens **I** nHome

Reply:
oooo I see